lib/pbrToMaterialsCommon.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+var Cesium = require('cesium');
+var addExtensionsRequired = require('./addExtensionsRequired');
+var ForEach = require('./ForEach');
+var getJointCountForMaterials = require('./getJointCountForMaterials');
+var removeExtensionsUsed = require('./removeExtensionsUsed');
+
+var defined = Cesium.defined;
+
+module.exports = pbrToMaterialsCommon;
+
+/**
+ * Convert PBR materials to use a KHR_materials_common BLINN shader.
+ * This is a lossy conversion, and as such should only be used for
+ * compatibility reasons. Normal texture and metallic roughness information
+ * will be lost, and some color or texture information may be lost.
+ *
+ * @param {Object} gltf A javascript object containing a glTF asset.
+ * @returns {Object} The glTF asset with materials converted to KHR_materials_common
+ */
+function pbrToMaterialsCommon(gltf) {
+    var materialJointCount = getJointCountForMaterials(gltf);
+    ForEach.material(gltf, function(material, materialId) {
+        var values = {
+            ambient : [ 0.0, 0.0, 0.0, 1.0 ],
+            diffuse : [ 0.0, 0.0, 0.0, 1.0 ],
+            emission : [ 0.0, 0.0, 0.0, 1.0 ],
+            specular : [ 0.0, 0.0, 0.0, 1.0],
+            shininess : [ 0.0 ]
```

Comment:
Eh, that's massively out of date... I don't really have a good answer other than that glTF 2.0 materials pre-PBR required all values to be arrays. I will put this in line with what is there currently, but it will probably change to something closer to this when it gets updated.